components/brave_new_tab/resources/components/background_caption.tsx

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import Icon from '@brave/leo/react/icon'
+
+import formatMessage from '$web-common/formatMessage'
+import { useLocale } from './locale_context'
+import { useNewTabState } from './new_tab_context'
+
+import {
+  BraveBackground,
+  SponsoredImageBackground } from '../models/new_tab_model'
+
+import { style } from './background_caption.style'
+
+function BraveBackgroundCredits(
+  props: { background: BraveBackground}
+) {
+  const { getString } = useLocale()
+  const { author, link } = props.background
+  if (!author) {
+    return null
+  }
+  return (
+    <a
+      className='photo-credits'
+      href={link || '#'}
```

Comment:
less concerned about this location since these are rarely coming from any place other than something provided by us. However if you're writing a util function to sanitize the URL for the one below might as well use it here too.

Reply:
Fixed